### ISSUE TITLE
Release v3.25.0 - Broken link - Changelog v3.16

### ISSUE THREAD

**Issue by cafferata** [StackExchange/dnscontrol#1985]
With the merge of #1809, the v3.16 changelog URL (within the [Deprecation warnings](https://github.com/StackExchange/dnscontrol/releases/tag/v3.25.0#:~:text=Deprecation%20warnings) section) in the latest release [v3.25.0](https://github.com/StackExchange/dnscontrol/releases/tag/v3.25.0) (_from yesterday_) is broken. I think it's good to fix it since these release notes have just been published.

```diff
-https://github.com/StackExchange/dnscontrol/blob/v3.25.0/v316.md
+https://docs.dnscontrol.org/release/v316
```

cc: @tlimoncelli @cdhunt

**Comment by cafferata** [comment]
Cross reference https://github.com/StackExchange/dnscontrol/issues/1967 https://github.com/StackExchange/dnscontrol/issues/1985.

**Comment by tlimoncelli** [comment]
I have fixed the release notes for 3.25.0
